packages/nextra/src/mdx-plugins/static-image.ts

```diff
@@ -6,6 +6,8 @@ import { truthy } from '../utils'
 import { existsSync } from '../file-system'
 import { EXTERNAL_URL_REGEX, PUBLIC_DIR } from '../constants'
 
+const blurFormat = [".jpeg", ".png", ".webp", ".avif"]
```

Comment:
Maybe `.jpg` too?

Reply:
i got this extension array from here https://github.com/vercel/next.js/blob/6cfebfb02c2a52a1f99fca59a2eac2d704d053db/packages/next/client/image.tsx#LL702C82-L702C99